modules/carto/bundle.ts

```diff
@@ -7,4 +7,7 @@ if (!deck.LineLayer) {
   throw new Error('@deck.gl/layers is not found');
 }
 
+// Export carto layer library for pydeck integration
+globalThis.CartoLayerLibrary = {CartoLayer: CartoUtils.CartoLayer};
```

Comment:
I must say I'm a bit lost as to why this is necessary for pydeck. Why doesn't pydeck need similar exports to use other deck Layers?